models/search_must_fields_test.go

```diff
@@ -73,3 +73,126 @@ func (ss *SearchSuite) TestSearchTransactionsWithMustFields() {
 	transactions, _ = results.([]*TransactionResult)
 	assert.Equal(t, 0, len(transactions), "No transaction should exist for given query")
 }
+
+func (ss *SearchSuite) TestSearchAccountsWithFieldOperators() {
```

Comment:
Why are test cases method? Can we use test utils instead?

Reply:
Please make sure a test case test one functionality. Name the test case in that way as well.